rust/candid/src/parser/types.rs

```diff
@@ -1,22 +1,33 @@
 use crate::types::Label;
 use crate::Result;
 use pretty::RcDoc;
+use crate::{CandidType, Deserialize};
```

Comment:
@chenyan-dfinity I tried to use `CandidType` below each place where I added `Deserialize`, but I got some macro errors.  I think the issue is about the macro being used from the crate that defines it?